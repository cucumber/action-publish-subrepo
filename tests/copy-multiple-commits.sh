#!/bin/bash

pushd monorepo || exit
mkdir ruby
echo "# Ruby project" >ruby/README.md
git add .
git commit --message "Add ruby"

mkdir php
echo "# PHP project" >php/README.md
git add .
git commit --message "Add php"

echo "# An amazing PHP project" >php/README.md
git add .
git commit --message "Update php readme"

# run the tool
publish-to-subrepo php "$SUBREPO"
