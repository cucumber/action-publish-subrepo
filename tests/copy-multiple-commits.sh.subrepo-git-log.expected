Update php readme  (HEAD -> main)
Add php 
